topostats/run_topostats.py

```diff
@@ -129,6 +129,7 @@ def process_scan(
     img_path_px2nm: Dict[str, Union[np.ndarray, Path, float]],
     base_dir: Union[str, Path],
     filter_config: dict,
+    scars_config: dict,
```

Comment:
If the `scars_config` section is going to remain as an independent top-level dictionary in `default_config.yaml` then this will be required.

But there is the option of moving the whole `remove_scars` configurations and nesting it under the `filter:` section.  If this is going to be done then you wouldn't need to pass `scars_config` in as a separate argument to `process_scan` as it is nested within `filter_config`.